modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/expected/upgrade/UpgradeJavaGetPortletGroupIdMethodCheck.testjava

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.source.formatter.dependencies.upgrade;
+
+import com.liferay.portal.kernel.theme.ThemeDisplay;
+
+/**
+ * @author Tamyris Bernardo
+ */
+public class UpgradeJavaGetPortletGroupIdMethodCheck {
+
+	public void method(){
+
+		ThemeDisplay themeDisplay = LiferayUtil.getThemeDisplay(request);
+
+		long groupId = themeDisplay.getScopeGroupId();
+	}
```

Comment:
Please format the test files. I know with the upgrade checks, formatting guidelines for the test inputs are fairly loose. However, we still need to format the parts of the test files that are not relevant for the check. In this case, it is the extraneous newlines, the missing space between `method` and `()`, and t he missing newline character after the ending brace of `method()`.